LICENSE

```diff
@@ -187,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014 Google Inc.
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
Why is this replaced with the placeholders? @jba who should probably be the reviewer, not me.